config/config.json

```diff
@@ -153,6 +153,9 @@
         "AboutLink": "https://about.mattermost.com/default-about/",
         "HelpLink": "https://about.mattermost.com/default-help/",
         "ReportAProblemLink": "https://about.mattermost.com/default-report-a-problem/",
+        "AdministratorsGuideLink": "https://about.mattermost.com/administrators-guide/",
```

Comment:
I believe they shouldn't, because they're also hardcoded in `model/config.go`, which is what's used if it can't be found in `config.json`